### PR TITLE
(Menu) Added toggle menu option for "Down+Select"

### PR DESCRIPTION
**Pull request by Tony556**
Inspired and based off the changes made in https://github.com/libretro/RetroArch/pull/7774

This is a common shortcut used by 8BitDo and (S)NES Classic modders, and I feel would be appreciated by other users.

### CODE REVIEW

**Comment by orbea** [comment]
Can you please squash your commits?

File: menu/menu_setting.c [hunk 1]
```diff
@@ -2025,6 +2025,9 @@ static void setting_get_string_representation_toggle_gamepad_combo(
       case INPUT_TOGGLE_HOLD_START:
          strlcpy(s, msg_hash_to_str(MENU_ENUM_LABEL_VALUE_HOLD_START), len);
          break;
+      case INPUT_TOGGLE_DOWN_SELECT:
+         strlcpy(s, "Down + Select", len);
```
Comment (RobLoach) [inline menu/menu_setting.c]: Should likely be localized via `msg_hash_to_str(MENU_ENUM_LABEL_VALUE_DOWN_SELECT)` or similar.

**Comment by Tony556** [comment]
@orbea Squashed!

@RobLoach Added the intl compatibility!

**Comment by RobLoach** [comment]
@Tony556 Resolved the merge conflicts, hope you don't mind :wink:

**Comment by orbea** [comment]
@RobLoach I would suggest `git rebase` instead of `git merge` here. :)

Try something like this on the branch you want to rebase against an upstream remote.
```
# Add the upstream remote.
git remote add upstream https://github.com/libretro/RetroArch

# verify the current remote repos
git remote -v

# fetch changes
git fetch upstream

# and rebase against the upstream remote
git rebase -i upstream/master

# push the changes to your branch
git push --force origin branch_name
```
This will take you to an interactive editor where you modify the commits from `pick` to one of the other options explained in the comments below such as `squash` or `fix`. You can also reorder commits or remove them.

For example.
```
pick 0f8de9b3b3 makefile.common: Clean up SDL conditionals.

# Rebase 158cec17fd..0f8de9b3b3 onto 158cec17fd (1 command)
#
# Commands:
# p, pick <commit> = use commit
# r, reword <commit> = use commit, but edit the commit message
# e, edit <commit> = use commit, but stop for amending
# s, squash <commit> = use commit, but meld into previous commit
# f, fixup <commit> = like "squash", but discard this commit's log message
# x, exec <command> = run command (the rest of the line) using shell
# d, drop <commit> = remove commit
# l, label <label> = label current HEAD with a name
# t, reset <label> = reset HEAD to a label
# m, merge [-C <commit> | -c <commit>] <label> [# <oneline>]
# .       create a merge commit using the original merge commit's
# .       message (or the oneline, if no original merge commit was
# .       specified). Use -c <commit> to reword the commit message.
#
# These lines can be re-ordered; they are executed from top to bottom.
#
# If you remove a line here THAT COMMIT WILL BE LOST.
#
#       However, if you remove everything, the rebase will be aborted.
#
#
# Note that empty commits are commented out
```

**Comment by RobLoach** [comment]
Done! Thanks for the #protips. Your git-fu is strong.

**Pull request merged**